rtcweb-overview.xml

```diff
@@ -431,8 +431,8 @@
 
       <t>If the two Web servers are operated by different entities, the
       inter-server signaling mechanism needs to be agreed upon, either by
-      standardization or by other means of agreement. Existing protocols (for
-      example SIP <xref target="RFC3261"/> or XMPP <xref target="RFC6120"/>)
+      standardization or by other means of agreement. Existing protocols
+      (e.g. SIP <xref target="RFC3261"/> or XMPP <xref target="RFC6120"/>)
```

Comment:
I have a principled objection to using Latin-based abbreviations when there are common English words that mean exactly the same thing. It provides a stumbling block for international readers. (I had to look up "e. g." to make sure there was no difference in meaning - which there isn't, according to Wiktionary).